### DISCUSSION TITLE
Where is the `Files changed` tab on the Communicate using Markdown training, the third unit?

### DISCUSSION THREAD

**Discussion by rfalanga** [skills/.github#635] [Communicate using Markdown] [answered]
Following the instructions in the third unit of the *Communicate using Markdown* training it says:

<img width="1178" alt="image" src="https://github.com/user-attachments/assets/4f815514-7e8e-4082-a456-bb26acf8ae6d">

There is no `Files changed` tab?! Here's what I've got:

<img width="769" alt="image" src="https://github.com/user-attachments/assets/71fa2061-c06c-4083-a787-b0410be6e1f9">

I don't see a `Files changed` tab there at all. I am confused and lost. Please, help.

**Comment by dayane112** [comment, marked as answer]
You have to select create pull request again on the next page that loads and on the following page you will see the file change tab.

**Reply by rfalanga** [reply]
Thank you very much!!

**Reply by Jbriggs5635** [reply]
What new in your end
